README.md

```diff
@@ -10,7 +10,22 @@ Our stretch goals include adding collaborators to any bucket list, viewing popul
 
 ## Project task requirements
 
-*placeholder*
+### Minimal requirements (will definitely complete)
```

Comment:
I think there are several points missing from minimal:
Check off destination in a bucket
View destinations in a bucket on a map with markers marking each destination
Hover on pins to see destination digest, click to see full detail

Reply:
I was thinking whether we wanted to put map in because it's a bit heavy.. Maybe i will put it there too.

Reply:
Yea, i think we should place this in minimal (since we are definitively completing it by the end of this project)